### PR TITLE
add-section-hero

### PR DESCRIPTION
**Pull request by Maksimka66**

### CODE REVIEW

File: src/partials/hero.html [hunk 1]
```diff
@@ -1,3 +1,27 @@
 <section class="hero">
-
-</section>
\ No newline at end of file
+  <div class="hero-container">
```
Comment (SergGrishkov) [inline src/partials/hero.html]: After tag `section ` should be `<div class="hero-container container">`. In container should be added all your code.

File: src/partials/hero.html [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+    </h1>
+    <button class="hero-btn-order" type="button">
+      Order
+      <svg class="hero-arrow" width="26" height="10">
+        <use href="/src/img/hero/hero-arrow.svg#icon-hero-arrow"></use>
+      </svg>
+    </button>
+    <img
+      class="hero-img"
+      srcset="/src/img/hero/girl@2x.jpg 2x"
```
Comment (SergGrishkov) [inline src/partials/hero.html]: Shoul be change path to file

File: src/partials/hero.html [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+      src="/src/img/hero/girl.jpg"
```
Comment (SergGrishkov) [inline src/partials/hero.html]: Shoul be change path to file

File: src/partials/hero.html [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+      alt="Satisfied client with our production"
+      width="392"
+      height="545"
+    />
+  </div>
+  <div class="hero-embroidery">
+    <svg class="hero-star" width="52" height="51">
+      <use href="/src/img/hero/arrow-and-star.svg#icon-hero-star"></use>
```
Comment (SergGrishkov) [inline src/partials/hero.html]: Shoul be change path to file

File: src/partials/hero.html [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
-</section>
\ No newline at end of file
+  <div class="hero-container">
+    <h1 class="hero-header">
+      Reviving the traditional Ukrainian artistry in every stitch.
+    </h1>
+    <button class="hero-btn-order" type="button">
+      Order
+      <svg class="hero-arrow" width="26" height="10">
+        <use href="/src/img/hero/hero-arrow.svg#icon-hero-arrow"></use>
```
Comment (SergGrishkov) [inline src/partials/hero.html]: Shoul be change path to file

**Review by SergGrishkov — CHANGES_REQUESTED** [review]

File: src/partials/hero.html (hunk 1 above)
Comment (Maksimka66) [reply src/partials/hero.html]: Done

File: src/partials/hero.html (hunk 2 above)
Comment (Maksimka66) [reply src/partials/hero.html]: Done

File: src/partials/hero.html (hunk 3 above)
Comment (Maksimka66) [reply src/partials/hero.html]: Done

File: src/partials/hero.html (hunk 4 above)
Comment (Maksimka66) [reply src/partials/hero.html]: Done

File: src/partials/hero.html (hunk 5 above)
Comment (Maksimka66) [reply src/partials/hero.html]: Done

**Review by SergGrishkov — APPROVED** [review]

**Pull request merged**